src/App.js

```diff
@@ -0,0 +1,24 @@
+import { Provider } from 'react-redux';
+import AppRoutes from './routes';
+import store from './redux/store';
+import NavBar from './components/navbar/NavBar';
+import './media-queries/Small.css';
+import './media-queries/Large.css';
+import './media-queries/XLarge.css';
+import './media-queries/XXLarge.css';
+import './media-queries/Desktop.css';
+import MobileNavBar from './components/navbar/MobileNavBar';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

That way, your functions will meet the latest standards. 💪